scss/_custom-styles.scss

```
@@ -246,3 +246,9 @@
     column-count: 4;
   }
 }
+
+// >> Border thick utility.
+
+.border-thick {
+  border-width: .25rem;
```

Something like this should make the stylelint checks happy, unless a better solution exists.
```suggestion
  /* stylelint-disable-next-line declaration-no-important */
  border-width: .25rem !important;
```

---

I think if we don't want to import the whole _border.scss file then we should make a file for _custom_overrides like we did for ua-bootstrap 

https://bitbucket.org/uadigital/ua-bootstrap/src/master/src/sass/_custom_overrides.scss

---

Or is this a new class?